compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/reference/PsiToAnnotationSpecHelper.kt

```diff
@@ -76,22 +75,20 @@ internal fun KtExpression.codeBlock(module: ModuleDescriptor): CodeBlock {
 }
 
 private fun FqName.asMemberName(module: ModuleDescriptor): MemberName {
-  val segments = pathSegments().map { it.asString() }
-  val simpleName = segments.last()
-  val prefixFqName = FqName.fromSegments(segments.dropLast(1))
-  return prefixFqName.asClassNameOrNull(module)?.let {
-    val classOrObj = prefixFqName.toClassReferencePsiOrNull(module)?.clazz
```

Comment:
I was debugging this code for another issue and saw the opportunity to remove this deprecated call.